seqr/utils/middleware.py

```diff
@@ -33,7 +33,7 @@ class JsonErrorMiddleware(MiddlewareMixin):
     @staticmethod
     def process_exception(request, exception):
         if request.path.startswith('/api'):
-            exception_json = {'message': str(exception)}
+            exception_json = {'message': exception.args[0]}
```

Comment:
I'm not sure I like this change. I thought `str(exception)` was valid in python 3? Its entirely possible that there are subclasses of exceptions from thrid parties that implement there own string method and don't take args...

Reply:
`str(exception)` does work in Python 3.